src/view/screens/Feeds.tsx

```diff
@@ -433,7 +435,7 @@ export function FeedsScreen(_props: Props) {
                     href="/settings/saved-feeds"
                     accessibilityLabel={_(msg`Edit My Feeds`)}
                     accessibilityHint="">
-                    <CogIcon strokeWidth={1.5} style={pal.icon} size={28} />
+                    <Gear style={pal.textLight} size="lg" />
```

Comment:
Should be `pal.icon`, otherwise there's a mismatch with the icon next to it

<img width="360" alt="Screenshot 2024-04-08 at 15 58 21" src="https://github.com/bluesky-social/social-app/assets/10959775/f46848c6-db98-421a-b6fe-57c34ffdf743">